java-grafeas/samples/install-without-bom/pom.xml

```diff
@@ -1,84 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.google.cloud</groupId>
-  <artifactId>-install-without-bom</artifactId>
-  <packaging>jar</packaging>
-  <name>Google Grafeas Install Without Bom</name>
-  <url>https://github.com/googleapis/java-grafeas</url>
-
-  <!--
-    The parent pom defines common style checks and testing strategies for our samples.
-    Removing or replacing it should not affect the execution of the samples in anyway.
-  -->
-  <parent>
-    <groupId>com.google.cloud.samples</groupId>
-    <artifactId>shared-configuration</artifactId>
-    <version>1.2.0</version>
-  </parent>
-
```

Comment:
Seems odd that these samples got added in again. I thought we had already deleted the non-generated samples for all modules.

Reply:
Nvm looks like owlbot exclusion list was updated 5 days ago and the commit went in 6 days ago. Should be fine now.